src/metax_api/tests/api/rest/base/views/directories/write.py

```diff
@@ -54,3 +52,129 @@ def _get_second_new_test_data(self):
             "identifier": "urn:nbn:fi:csc-ida201401200000000002",
         })
         return from_test_data
+
+
+class DirectoryApiWriteTests(DirectoryApiWriteCommon):
```

Comment:
v2 api used here. This test class should be copied to v2 tests as is and change the apis used here to v1

Reply:
weird that i didn't... too hurry to PR :)